doc/config.yaml

```diff
@@ -16,18 +16,22 @@ params:
   hero:
     title: Hugo Theme
     subtitle: Used by core projects in the scientific Python ecosystem.
-    buttontext: Go to docs
+    buttontext: Get started
```

Comment:
I just had a stop when reading `Get started` at the top of the page and then `Getting started` as a section. Maybe we should rename one or the other.

Reply:
In b87e168, I changed ` ## Getting started ` to ` ## Download and install` in `doc/content/getstarted.md`.